rpc-server.go

```diff
@@ -128,7 +128,10 @@ func (s *storageServer) RenameFileHandler(arg *RenameFileArgs, reply *GenericRep
 // Initialize new storage rpc.
 func newRPCServer(exportPath string) (*storageServer, error) {
 	// Initialize posix storage API.
-	storage, err := newPosix(exportPath)
+	// Ignore disk errors and successfully initialize posix, errors
+	// will be handled by top level caller.
+	ignoreDiskErr := true
+	storage, err := newPosix(exportPath, ignoreDiskErr)
```

Comment:
I think it would be better to handle error and have ignoreDiskErr here than at newPosix()